src/static/css/pages/sub-pages.less

```diff
@@ -3,10 +3,51 @@
    Sub-Pages pages at /sub-pages/*.
    ========================================================================== */
 
+.green-bullets {
```

Comment:
Sorry to inform you you're reinventing the wheel here :(

https://cfpb.github.io/cf-typography/docs/#branded-list-modifier